mps_youtube/helptext.py

```diff
@@ -238,6 +238,9 @@ def helptext():
     {2}set video_format <auto|mp4|webm|3gp>{1} - set default music video format
     {2}set api_key <key>{1} - use a different API key for accessing the YouTube Data API
     {2}set set_title true|false{1} - change window title
+    
+    {2}Additionally, set -t may be used to temporarily change a setting without
+    {2}saving it to disk
```

Comment:
Any specific reason for the whitespace? Why make it two line instead of one?

Also, I think you have to end the color using `{1}`.

Reply:
I make it two lines instead of one to adhere to the 80 chars / column rule that seems to be used in the rest of the document.